front-end/src/main/kotlin/org/jetbrains/kotlin/spec/viewer/MarkUpArranger.kt

```diff
@@ -0,0 +1,49 @@
+package org.jetbrains.kotlin.spec.viewer
+
+import js.externals.jquery.JQuery
+import js.externals.jquery.`$`
+import org.jetbrains.kotlin.spec.utils.format
+import org.w3c.dom.HTMLElement
+
+
+object MarkUpArranger {
+    fun showMarkUpForParagraphs(paragraphsInfo: List<Map<String, Any>>, sectionsPath: String) {
+        paragraphsInfo.forEachIndexed { paragraphIndex, paragraph ->
+            val paragraphNumber = paragraphIndex + 1
+            val paragraphEl = `$`(paragraph["paragraphElement"] as HTMLElement).apply { addClass("with-tests") }
```

Comment:
Use safe cast to avoid cast exceptions at runtime

Reply:
done